src/main/java/Duke.java

```diff
@@ -1,10 +1,115 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static String logo = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+
+    private static Task[] tasks = new Task[100];
+    private static int taskCount = 0;
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
+        printGreeting(logo);
+
+        Scanner in = new Scanner(System.in);
+
+        while (true){
+            String line = in.nextLine();
+
+            if (line.equals("list")){
+                printList();
+                continue;
+            }
+            else if (line.equals("bye")) {
```

Comment:
Consider putting else if statement on the same line just after the previous curly bracket.